plugins/processors/t128_filter/t128_filter.go

```diff
@@ -0,0 +1,129 @@
+package t128_filter
+
+import (
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+const sampleConfig = `
+[[processors.t128_filter]]
+  ## The conditions that must be met to pass a metric through. This is similar
+  ## behavior to a tagpass, but the multiple tags are ANDed
+  [[processors.t128_filter.condition]]
+
+  [processors.t128_filter.condition.tags]
+     #tag1 = ["value1", "value2"]
```

Comment:
do we need to do some kind of validation to ensure the same `tag` doesn't appear twice in the same condition or does TOML guarantee that. In other words this config should result in an error:

```
  [processors.t128_filter.condition.tags]
     #tag1 = ["value1", "value2"]
     #tag1 = ["value3"]
```

Reply:
Yea we get that for free from the toml unmarshal. I'll add a testcase to prove